### DISCUSSION TITLE
Opening modal on top of each other

### DISCUSSION THREAD

**Discussion by kadiraydinli** [gorhom/react-native-bottom-sheet#422] [Q&A]
Hello, how can I open more than one mod in a row? While a modal is open, I open the other modal with `.present()`, but the previously open modal closes.

I'm using:

`react-native@0.63.4`
`@gorhom/bottom-sheet@2.3.0`

**Comment by bardonadam** [comment]
Anybody managed to do this?

**Comment by ilyagru** [comment]
`stackBehavior="push"`
